### PR TITLE
feat: Added validation for iconType

### PR DESCRIPTION
**Pull request by Lolly1150**
Added validation for iconType

### CODE REVIEW

File: packages/validation/src/schemas/anchors.ts [hunk 1]
```diff
@@ -39,5 +39,6 @@ export const anchorsSchema = z
         invalid_type_error: "Version must be a string in the versions array.",
       })
       .optional(),
+      iconType: z.enum(["brands", "duotone" , "light" , "sharp-solid" , "solid" , "thin"], { invalid_type_error: "anchor iconType must be one of brands, duotone, light, sharp-solid, solid, thin"}).optional(),
```
Comment (tobias-mintlify) [inline packages/validation/src/schemas/anchors.ts:42]: Can you add a unit test expecting an error when iconType is not one of these?

**Review by tobias-mintlify — CHANGES_REQUESTED** [review]

File: packages/validation/src/schemas/anchors.ts (hunk 1 above)
Comment (Lolly1150) [reply packages/validation/src/schemas/anchors.ts:42]: In what file for test

Comment (tobias-mintlify) [reply packages/validation/src/schemas/anchors.ts:42]: `packages/validation/tests/schemas/anchorsSchema.test.js`

File: client/src/types/config.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export type Anchor = {
   color?: string | Gradient;
   isDefaultHidden?: boolean;
   version?: string;
+  iconType?: 'brands' | 'duotone' | 'light' | 'regular' | 'sharp-solid' | 'solid' | 'thin'
```
[the remark this answers is not in the document]
Comment (hahnbeelee) [reply client/src/types/config.ts:36]: You might want to make this a type and store it in the `types` folder. It's also being used in `src/ui/Icon.tsx`.
Can you create a type and replace it here and in Icon.tsx?

File: packages/validation/src/schemas/anchors.ts (hunk 1 above)
Comment (Lolly1150) [reply packages/validation/src/schemas/anchors.ts:42]: Sure

**Review by tobias-mintlify — APPROVED** [review]
Congrats on your first contribution! Next steps are to pass the property to the Icon component from `AnchorLink`.

**Comment by Lolly1150** [comment]
Thankyou, can you create issue to describe this?

**Comment by Lolly1150** [comment]
And the code that will do for this should I push it here or on another pr?

**Comment by tobias-mintlify** [comment]
You can make a separate PR. I'll merge this one in

**Comment by Lolly1150** [comment]
Ok, can you create issue to describe this?

**Pull request merged**